.ci/travis-checks.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [[ $TRAVIS_OS_NAME == "linux" ]]; then
+  if [[ $BUILDTYPE == "Debug" ]]; then   ## TODO: generalize h cpp files search to top-directory? run bash script by zach here?
+    echo "Checking code style..."
+    clang-format -i \
```

Comment:
We should probably update this to just run the clangify file?

Reply:
Or, at least, copy what it does in terms of file coverage

Reply:
I defensively look into running your script there! 👍